docs/dao-setup.md

```diff
@@ -56,6 +56,13 @@ rpcuser=YOUR_USER_NAME
 rpcpassword=YOUR_PW
 blocknotify="%AppData%\Bitcoin\blocknotify.bat" %s
 ```
+Note: If you are using bitcoin core version 0.19.0.1
+should add following two lines to the config file before rpcport option.
+```
+[regtest]
+peerbloomfilters=1
```

Comment:
I believe this option has been available for some time now, but just recently changed to being disabled by default. Therefore, it should be safe to add this parameter regardless if you are still using an older bitcoin core. So rather than adding this as a separate note, just add it into the examples above.

Reply:
Have added into the examples above now. @devinbileck